includes/processor_admin.form.inc

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * @file
+ * Processor admin form.
+ */
+
+/**
+ * Form building function.
+ */
+function islandora_xquery_processor_admin_form(array $form, array &$form_state) {
+  form_load_include($form_state, 'inc', 'islandora_xquery', 'includes/processor_admin.form');
+
+  $form_state['storage']['implementations'] = IslandoraXquery::getImplementations();
+
+  try {
+    $default_value = IslandoraXquery::getImplementation()[0];
+  }
+  catch (IslandoraXqueryNoExecutableException $e) {
+    // The one configured, even though it doesn't appear to be executable...
+    $default_value = $e->getImplementationInfo()[0];
+  }
+
+  $form['#tree'] = TRUE;
+
+  $form['processors'] = array(
+    '#type' => 'radios',
+    '#title' => t('Processors'),
+    '#options' => drupal_map_assoc(array_keys($form_state['storage']['implementations'])),
+    '#default_value' => $default_value,
+  );
+
+  $form['processor'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Processor config'),
+  );
+
+  foreach ($form_state['storage']['implementations'] as $name => $class) {
+    $form['processor'][$name] = (is_callable(array($class, "getForm")) ?
+      call_user_func_array(array($class, "getForm"), array(array('processor', $name), &$form_state)) :
```

Comment:
40 | ERROR | If the line declaring an array spans longer than 80
    |       | characters, each element should be broken into its own
    |       | line